packages/bbui/src/Label/Label.svelte

```diff
@@ -9,6 +9,7 @@
 
 <TooltipWrapper {tooltip} {size}>
   <label
+    data-testid="label"
```

Comment:
It strikes me as a bit strange that `testing-library` out of the box uses it's own attribute for selection instead of giving you the option to query by `id`, which I've seen other ui testing libraries use. 

If we did want to use `id` instead of adding an attributes dedicated to tests if seems possible using this: https://testing-library.com/docs/queries/bytestid/#overriding-data-testid

Reply:
I think a lot of the more "modern" testing approaches recommend this, or at least I've seen Jest and Cypress recommend it also. I think the idea is that having a totally separate attribute to query on is the most resilient to change, as ids actually have a use outside of the test for stuff like inputs so it's technically relying on an implementation detail 🤷